Expecto/Expect.fs

```diff
@@ -304,8 +304,13 @@ let stringStarts subject prefix message =
       "%s. Expected subject string to start with the prefix. Differs at position %i with subject '%c' and prefix '%c'.%s"
       message i s p (printVerses " prefix" prefix "subject" subject)
 
-/// Expects the two values to equal each other.
-let equal (actual : 'a) (expected : 'a) message =
+let colorizeString (color:ConsoleColor) text : string =
```

Comment:
Why is this public?

Reply:
Yea, that's a helper. I'll make it private